modules/overview-of-kueue-resources.adoc

```diff
@@ -0,0 +1,136 @@
+
+* Homogeneous resources are identical across the cluster: same node type, CPUs, memory, accelerators, and so on.
+* Heterogeneous resources have variations across the cluster.
+
+If a cluster has homogeneous resources, or if it is not necessary to manage separate quotas for different flavors of a resource, the cluster administrator can create an empty `ResourceFlavor` object named `default-flavor`, without any labels or taints, as follows:
+
+.Empty Kueue resource flavor for homegeneous resources
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: default-flavor
+----
+
+If a cluster has heterogeneous resources, the cluster administrator can define a different resource flavor for each variation in the resources available. 
+Example variations include different CPUs, different memory, or different accelerators.
+The cluster administrator can then associate the resource flavors with cluster nodes by using labels, taints, and tolerations, as shown in the following example.
+
+.Example Kueue resource flavor for heterogeneous resources
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "spot"
+spec:
+  nodeLabels:
+    instance-type: spot
+  nodeTaints:
+  - effect: NoSchedule
+    key: spot
+    value: "true"
+  tolerations:
+  - key: "spot-taint"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+----
+
+For more information about configuring resource flavors, see link:https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/[Resource Flavor] in the Kueue documentation.
+
+
+== Cluster queue
+
+The Kueue `ClusterQueue` object manages a pool of cluster resources such as pods, CPU, memory, and accelerators. 
+A cluster can have multiple cluster queues, and each cluster queue can reference multiple resource flavors.
+
+The cluster administrator can configure a cluster queue to define the resource flavors that the queue manages, and assign a quota for each resource in each resource flavor.
+The cluster administrator can also configure usage limits and queueing strategies to apply fair sharing rules across multiple cluster queues in the cluster.
+ 
+The following example configures a cluster queue to assign a quota of 9 CPU, 36 GiB memory, 5 pods, and 5 NVIDIA GPUs.
+
+.Example cluster queue
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "pods", "nvidia.com/gpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 9
+      - name: "memory"
+        nominalQuota: 36Gi
+      - name: "pods"
+        nominalQuota: 5
+      - name: "nvidia.com/gpu"
+        nominalQuota: '5'
+----
+
+The cluster queue will start a distributed workload only if the total required resources are within these quota limits. 
```

Comment:
Could the use of "will" here be avoided?

 The cluster queue starts a distributed workload only if the total required resources are within these quota limits.